static/js/containers/ChannelPage.js

```diff
@@ -16,41 +16,68 @@ import { getChannelName } from "../lib/util"
 
 import type { Dispatch } from "redux"
 import type { Match } from "react-router"
+import type { RestState } from "../flow/restTypes"
+import type { Channel, Post } from "../flow/discussionTypes"
 
 class ChannelPage extends React.Component {
   props: {
     match: Match,
     dispatch: Dispatch,
-    channels: Object,
-    postsForChannel: Object,
-    posts: Object
+    channels: RestState<Map<string, Channel>>,
+    postsForChannel: RestState<Map<string, Array<string>>>,
+    posts: RestState<Map<string, Post>>,
+    subscriptions: RestState<Array<Channel>>,
```

Comment:
Sounds good. Just renaming `subscriptions` is certainly good enough for now, but I'll put it another way: if we had the need to fetch all channels (not just the subscribed ones), what would we name that in the state if not `channels`? Just something to think about. I'm still getting used to redux hammock so maybe it's just lack of familiarity